### DISCUSSION TITLE
Vote for your favorite logo font and weight

### DISCUSSION THREAD

**Discussion by hexsecs** [uptane/uptane.github.io#75] [Polls]
Vote for your favorite

### Montserrat - Light
![image](https://github.com/uptane/uptane.github.io/assets/89217603/e658bbcf-0106-4b59-b8c0-ab3d46378748)

### Montserrat - Normal
![uptane-horizontal-color-montserrat](https://github.com/uptane/uptane.github.io/assets/89217603/c5568f54-b05a-4157-94d4-34038ec5e932)

### Montserrat - Medium
![uptane-horizontal-color-montserrat-medium](https://github.com/uptane/uptane.github.io/assets/89217603/b3ae6fa7-3947-4bbb-b32d-81b3e5d1da37)

### Raleway - Regular 400
![uptane-horizontal-color](https://github.com/uptane/uptane.github.io/assets/89217603/15f25290-0c4b-44e5-82ac-8898588a1adb)

**Comment by trishankatdatadog** [comment]
Actually, I changed my mind from "Montserrat - Light" to "Raleway - Regular 400"...

**Reply by jhdalek55** [reply]
I rather like the Raleway regular 400 as well. Certainly would be my preference over the Monserrat Light.

**Reply by hexsecs** [reply]
If you already voted and changed your mind, you can click on the "Hide Results" and revote. Right now there are two verbal votes for Raleway, but none in the poll.

**Reply by jhdalek55** [reply]
Ok. I switched my vote to Raleway.

**Comment by Abhijay007** [comment]
closed as resolved
